d3ploy/solver.py

```diff
@@ -166,7 +169,17 @@ def get_asc_key_list(dicti):
     """
     key_list = [' '] * len(dicti.values())
     sorted_caps = sorted(dicti.values(), reverse=True)
+    # store previous indx list to prevent
```

Comment:
inline comments aren't needed if the code itself is clear. Since they don't get run or tested, such comments are destined to become out-of-date cruft. 

“Redundant comments are just places to collect lies and misinformation.” 
― Robert C. Martin, Clean Code: A Handbook of Agile Software Craftsmanship